### DISCUSSION TITLE
Unable to run lima on macOS Monterey (`Unsupported option \\\"gssapiauthentication\\\"\\r\\nkex_exchange_identification`)

### DISCUSSION THREAD

**Discussion by rausberg** [lima-vm/lima#1916] [General]
Anyone ?
Logfiles .lima/default/serial(v).log remains empty.
Qemu works when standalone (tested freedos).
Following "lima-vm.io/docs/usage/" but likely I'm missing something basic...
Installed lima from macports, "limactl start" yields messages:
---
? Creating an instance "default" Proceed with the current configuration
INFO[0015] QEMU binary "/opt/local/bin/qemu-system-x86_64" seems properly signed with the "com.apple.security.hypervisor" entitlement 
INFO[0015] Attempting to download the image              arch=x86_64 digest="sha256:5ad255d32a30a2cda9f0df19f0a6ce8d6f3c81b63845086a4cb5c43cf97fcb92" location="https://cloud-images.ubuntu.com/releases/23.04/release-20230810/ubuntu-23.04-server-cloudimg-amd64.img"
INFO[0015] Using cache "/Users/tor/Library/Caches/lima/download/by-url-sha256/e54a5ae9983387df7e867747dfd1d4b75b09df51588512504b67579b7b7b4f1f/data" 
INFO[0016] Attempting to download the nerdctl archive    arch=x86_64 digest="sha256:3f8c494e3c6a265fe2a3c41ef9d6bc859eeeb22095b3353d3558d8120833a23a" location="https://github.com/containerd/nerdctl/releases/download/v1.5.0/nerdctl-full-1.5.0-linux-amd64.tar.gz"
INFO[0016] Using cache "/Users/tor/Library/Caches/lima/download/by-url-sha256/72a70b017cb55385b4c43d2bcc2a0333094089f1da0739a34c51c647bf249674/data" 
INFO[0021] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/tor/.lima/default/serial*.log") 
INFO[0021] SSH Local Port: 60022                        
INFO[0021] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0031] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0119] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0206] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0294] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0382] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0470] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0558] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
FATA[0616] did not receive an event with the "running" status 
---

**Comment by AkihiroSuda** [comment]
Please check the following files, there might be some error:
- `~/.lima/default/ha.*.log`
- `~/.lima/default/serial*.log`

**Reply by rausberg** [reply]
~/.lima/default/serial*.log remains empty, zero bytes.
Thank you for your answer.
~/.lima/default/ha.stderr.log has two errors but I honestly dont know if those are benign or even what to expect as I could not find much documentation, possibly my google-fu is lacking... 
[ha.stderr.log](https://github.com/lima-vm/lima/files/12910764/ha.stderr.log)
[ha.stdout.log](https://github.com/lima-vm/lima/files/12910766/ha.stdout.log)

**Reply by AkihiroSuda** [reply]
```
{"level":"debug","msg":"stdout=\"\", stderr=\"command-line line 0: Unsupported option \\\"gssapiauthentication\\\"\\r\\nkex_ex
change_identification: read: Connection reset by peer\\r\\nConnection reset by 127.0.0.1 port 60022\\r\\n\", err=failed to exe
cute script \"ssh\": stdout=\"\", stderr=\"command-line line 0: Unsupported option \\\"gssapiauthentication\\\"\\r\\nkex_excha
nge_identification: read: Connection reset by peer\\r\\nConnection reset by 127.0.0.1 port 60022\\r\\n\": exit status 255","ti
me":"2023-10-15T23:40:57+02:00"}
```

This seems unusual.
Are you using a third party `ssh` binary? (`which ssh`; `ssh -V`)

**Reply by AkihiroSuda** [reply]
> ~/.lima/default/serial*.log remains empty, zero bytes.

This seems unusual too.
Does `limactl start --video` print something in the video screen?

**Reply by rausberg** [reply]
At first I ran default macOS ssh (/usr/bin/ssh, OpenSSH_8.6p1, LibreSSL 3.3.6). Because of the error messages I installed /opt/local/ssh from macports (OpenSSH_9.5p1, OpenSSL 3.1.3 19 Sep 2023) - there was no change for the better or worse.

limactl start --video made the qemu window appear for the first time !

Screenshot attached:
<img width="752" alt="Screenshot 2023-10-16 at 14 43 42" src="https://github.com/lima-vm/lima/assets/86931817/51e4c5cb-6fc0-4bea-815d-7c115c37ce2d">

**Comment by rausberg** [comment]
Oops, this time the last output by "limactl start default" was somewhat different:
FATA[0600] and not FATA[0616]

**Reply by AkihiroSuda** [reply]
The numbers there just mean "600 seconds" and "616 seconds" since the invocation of the command. Not an error code.

**Reply by rausberg** [reply]
Ahh, makes sense.

BTW, where can documentation like that be found ?

**Comment by rausberg** [comment]
Just upgraded to lima 0.18.0, unfortunately no change for the better.
Any ideas as to what I'm missing here ?

**Reply by AkihiroSuda** [reply]
Is there something unusual on your host?
e.g., 
- the host is running inside VM
- the host has some enterprise security software that conflicts with VM

**Reply by AkihiroSuda** [reply]
Also what if you install QEMU and Lima with `brew install --build-from-source` ?

**Reply by rausberg** [reply]
VirtualBox is installed but not running, Malwarebytes is running but turning it off does not make a difference.

I use MacPorts as I have had issues when trying Homebrew, do they coexist without problems ?

**Reply by rausberg** [reply]
I installed lima 0.18.0 and qemu 8.0.5 from macports sources but unfortunately no improvements.

**Reply by rausberg** [reply]
I moved macports out of the way and installing lima from homebrew indeed works!
So there must be something odd with how macports configures lima and/or qemu.
Guess I'll file a ticket with macports...

Thank you for your help!
